lib/gui/pages/main/styles/_main.scss

```diff
@@ -106,3 +106,7 @@
 .page-main .step-size {
   color: $palette-theme-dark-disabled-foreground;
 }
+
+.page-main .popover-content {
+  -webkit-user-select: auto;
```

Comment:
Can you explain why this was needed? Also, it seems that this should be moved to `_bootstrap.scss` (without `.page-main`) so it gets applied to all popovers on the app.